packages/manager/modules/pci/package.json

```diff
@@ -17,8 +17,9 @@
     "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-pci' --include-filtered-dependencies -- yarn run dev:watch"
   },
   "dependencies": {
+    "@ovh-ux/manager-config": "^0.2.0",
     "@ovh-ux/ng-ovh-cloud-universe-components": "^1.1.0-alpha.0",
-    "@ovh-ux/ng-ovh-user-pref": "^1.0.0",
+    "@ovh-ux/ng-ovh-payment-method": "^2.0.0",
```

Comment:
You have to keep `"@ovh-ux/ng-ovh-user-pref": "^1.0.0",` as a dependency.

Reply:
Oops... Bad conflicts resolution :-$